packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -569,11 +568,7 @@ export default defineComponent({
     openRenameDialog(item) {
       this.$_rename_trigger({ resources: [item] })
     },
-    openSharingSidebar(file) {
-      if (file.share?.shareType === ShareTypes.link.value) {
-        this.openWithPanel('links-item')
```

Comment:
Please keep the behaviour and use `sharing-item#linkShares` for links, `sharing-item#peopleShares` for remaining share types. That will scroll to the respective panel correctly.

Reply:
Is this already working/merged? Because it does not work for me

Reply:
Okay nvm, rebased 🤦